net/pfSense-pkg-Quagga_OSPF/files/usr/local/pkg/quagga_ospfd.inc

```diff
@@ -312,14 +337,24 @@ if [ -e /var/run/quagga/ospfd.pid ]; then
 	/bin/pkill -F /var/run/quagga/ospfd.pid
 	/bin/rm -f /var/run/quagga/ospfd.pid
 fi
+if [ -e /var/run/quagga/bgpd.pid ]; then
+	kill -9 `cat /var/run/quagga/bgpd.pid`
+	rm -f /var/run/quagga/bgpd.pid
+fi
 EOF;
 	$rc_file_start = <<<EOF
+/bin/mkdir -p {$quagga_config_base}
+/bin/mkdir -p /var/etc/quagga
```

Comment:
This line and the line above it effectively do the same thing, but neither are necessary as the PHP code handles it when writing out the configuration.